rust/routee-compass/src/app/search/search_app.rs

```diff
@@ -122,8 +122,8 @@ impl SearchApp {
                 );
                 Ok(SearchAppResult {
                     route,
-                    tree,
-                    search_start_time,
+                    tree: Some(tree),
```

Comment:
did we want to unravel this Option wrap per [this discussion](https://github.com/NREL/routee-compass/issues/114#issuecomment-1933067350)?